tests/map_helpers.cpp

```diff
@@ -21,6 +25,33 @@ void wipe_map_terrain()
     g->m.build_map_cache( 0, true );
 }
 
+// removes all items in a 1 radius area around a point
+void i_clear_adjacent( const tripoint &p )
+{
+    auto close_trip = g->m.points_in_radius( p, 1 );
+    for( const auto &trip : close_trip ) {
+        g->m.i_clear( trip );
+    }
+}
+
+// generates a thick forest overmap tile. sets the adjacent tiles to thick forests as well.
+void generate_forest_OMT( const tripoint &p )
```

Comment:
For a generic function in `map_helpers`, it probably makes more sense to pass the terrain type in here as an argument, rather than have it be hardcoded to forest.